### PR TITLE
Updated changelog to include 2.1.4.0 and 1.5.0.6

### PR DESCRIPTION
**Pull request by GayleNeumann**
-- added info for 2.1.4.0 and 1.5.0.6 Enterprise patch releases

### CODE REVIEW

File: app/enterprise/changelog.md [hunk 1]
```diff
@@ -264,6 +287,40 @@ Kong Enterprise 2.1.3.0 version includes 2.1.0.0 (beta) features, fixes, known i
...
+
+#### Kong Gateway
+
+* Fixed propagation of posted health check across Nginx workers.
+
+#### Plugins
+
+* Collector (`collector`, for Brain and Immunity) 
+  * Fixed bug caused when trying to call `select_all`.
+  * Fixed bug caused when trying to read a request body when unknowned how to parse.
```
Comment (tlblessing) [inline app/enterprise/changelog.md:318]: "unknowned" is not a word.

```suggestion
  * Fixed bug caused when trying to read a request body when parsing was unknown.
```

**Review by tlblessing — CHANGES_REQUESTED** [review]
grammar edit

**Comment by RobSerafini** [comment]
@mikefero  - Can you take a quick look at this changelog update to make sure it is complete?

**Review by RobSerafini — APPROVED** [review]
+1 With Terry's change suggestion

**Review by tlblessing — APPROVED** [review]

**Pull request merged**